# # Definitions of tags to detect
#
# ## General remarks
#
# - All length in meters
# - Ellipsis (...) signifies that the previous element can be repeated multiple times.
#
# ## Standalone tag definitions
# ### Remarks
#
# - name is optional
#
# ### Syntax
#
# standalone_tags:
#   [
#     {id: ID, size: SIZE, name: NAME},
#     ...
#   ]
standalone_tags:
  [
    #{id: 6, size: 0.168},
    # {id: 1, size: 0.168},
    # {id: 13, size: 0.168},
    # {id: 25, size: 0.168}
  ]
# ## Tag bundle definitions
# ### Remarks
#
# - name is optional
# - x, y, z have default values of 0 thus they are optional
# - qw has default value of 1 and qx, qy, qz have default values of 0 thus they are optional
#
# ### Syntax
#
# tag_bundles:
#   [
#     {
#       name: 'CUSTOM_BUNDLE_NAME',
#       layout:
#         [
#           {id: ID, size: SIZE, x: X_POS, y: Y_POS, z: Z_POS, qw: QUAT_W_VAL, qx: QUAT_X_VAL, qy: QUAT_Y_VAL, qz: QUAT_Z_VAL},
#           ...
#         ]
#     },

#     ...
#   ]
tag_bundles:
  [
     {
       #name: 'DARPA' #This will be assigned same name as target_frame_live from launch file
       layout:
         [
          #  {id: 1, size: 0.168, x: 0.002, y: 0.042, z: 0.329, qw: -0.499, qx: -0.495, qy: 0.494, qz: 0.512},
          #  {id: 13, size: 0.168, x: 0.003, y: -0.298, z: -0.001, qw: 0.506, qx: 0.530, qy: -0.489, qz: -0.474},
          #  {id: 25, size: 0.168, x: 0.001, y: 0.333, z: 0.001, qw: -0.494, qx: -0.491, qy: 0.502, qz: 0.513}
           {id: 1, size: 0.168, x: 0.002, y: 0.042, z: 0.329, qw: -0.5, qx: -0.5, qy: 0.5, qz: 0.5},
           {id: 13, size: 0.168, x: 0.003, y: -0.298, z: -0.001, qw: -0.5, qx: -0.5, qy: 0.5, qz: 0.5},
           {id: 25, size: 0.168, x: 0.001, y: 0.333, z: 0.001, qw: -0.5, qx: -0.5, qy: 0.5, qz: 0.5}
         ]
     }
  ]
